tests/system_tests_multicast.py

```diff
@@ -429,6 +438,137 @@ def test_999_check_for_leaks(self):
         self._check_for_leaks()
 
 
+class MulticastMeshTest(TestCase):
+    """
+    Verify the multicast forwarding logic across a mesh router
+    configuration
+    """
+    @classmethod
+    def setUpClass(cls):
+        """Start a router"""
+        super(MulticastMeshTest, cls).setUpClass()
+
+        def router(name, mode, extra):
+            config = [
+                ('router', {'mode': mode,
+                            'id': name,
+                            'workerThreads': W_THREADS}),
+                ('listener', {'role': 'normal',
```

Comment:
This listener is not necessary. It is already being added here - https://github.com/skupperproject/skupper-router/blob/main/tests/system_test.py#L701

Reply:
Discussed this issue a bit further with @ganeshmurthy.

Updated with Ganesh's advice: keep the 'listener' line here, since the test creates routers with role=inter-router; update test to not specify: maxFrameSize, linkCapacity.